mw39/Dockerfile

```diff
@@ -0,0 +1,153 @@
+# Copy of https://github.com/wikimedia/mediawiki-docker/edit/master/1.36/apache/Dockerfile with PHP and MW versions changed
+FROM php:8.0-apache as mw39
```

Comment:
Do you really need 8.0? I recommend switching to 8.1 when using MW 39. We are running MW + WB + WB extensions on PHP 8.1, so this is likely fine for you as well.

Reply:
Euh… no, and choosing 8.1 from the get-go, I would probably have saved myself quite some headaches with composer not respecting platform configuration, etc…

Reply:
Updated the `Dockerfile` and the `README`

Reply:
I advise against merging this just yet. I'm seeing some very weird behavior with the Mediawiki installation where it claims there is no `Localsettings.php` file. May not be related to this image, but let me make sure it isn't.